src/CrandallGridWorld/TeamBug.java

```diff
@@ -1,62 +1,59 @@
-package CrandallGridWorld;

```

Comment:
Why? I think it should stay how it is, so that it can be distributed as a jar file